config/prow/monitoring/mixins/Makefile

```diff
@@ -29,6 +29,8 @@ grafana-dashboards: install clean-dashboard-out
 		echo "Generating $${output} from $${input} ..."; \
 		jsonnet -J vendor/ -J lib/ "grafana_dashboards/$${input}" > "./dashboards_out/$${output}"; \
 	done
+	@cd vendor/node-mixin && make clean node_alerts.yaml node_rules.yaml dashboards_out
```

Comment:
Since i was not able to unify node-mixin under our `grafana_dashboards` tree, i'll do an out-of-tree build for the node-exporter dashboards.
Perhaps this could be improved.